data/common.yaml

```diff
@@ -4,6 +4,7 @@ vra_puppet_plugin_prep::vro_password:      'puppetlabs'
 vra_puppet_plugin_prep::vro_password_hash: '$1$Fq9vkV1h$4oMRtIjjjAhi6XQVSH6.Y.'
 vra_puppet_plugin_prep::manage_autosign:    true
 vra_puppet_plugin_prep::manage_localuser:   true
+vra_puppet_plugin_prep::manage_sshd:        true
```

Comment:
If this was optional, then it should it be defaulted to `false` as to not add compatibility problems?

Reply:
I’m just talking about backwards compatibility, not about making the module more compatible in new installations. 

Users of the module that upgrade the module will have a better time with it set to true by default. The module is fairly naive and serves mainly to codify the PE side setup requirements for vRA plugin, so I think this in be general spirit of this module.

Reply:
Great, thanks for the explanation!